apitest/src/test/java/bisq/apitest/method/payment/AbstractPaymentAccountTest.java

```diff
@@ -125,7 +145,13 @@ protected final void verifyEmptyForm(File jsonForm, String paymentMethodId, Stri
                 PAYMENT_ACCOUNT_FORM.toJsonString(jsonForm),
                 Object.class);
         assertNotNull(emptyForm);
-        assertEquals(PROPERTY_VALUE_JSON_COMMENTS, emptyForm.get(PROPERTY_NAME_JSON_COMMENTS));
+
+        if (false && paymentMethodId.equals("SWIFT_ID")) {
```

Comment:
Maybe a TODO if this is a placeholder for future swift test

Reply:
Done.  Commit 429e943a363d76c6a8953d2905c5ab33dee8f6bc